pillar/fluentd/cas.sls

```diff
@@ -9,28 +9,42 @@ fluentd:
       settings:
         - directive: source
           attrs:
-            - tag: cas.mitx.shibboleth.error
+            - '@id': cas_nginx_access_log
             - '@type': tail
             - enable_watch_timer: 'false'
-            - path: /var/log/apache2/mitx_shibboleth_error.log
-            - pos_file: /var/log/apache2/mitx_shibboleth_error.log.pos
-            - format: apache_error
+            - tag: cas.nginx.access
+            - path: /var/log/nginx/access.log
+            - pos_file: /var/log/nginx/access.log.pos
+            - nested_directives:
+                - directive: parse
+                  attrs:
+                    - '@type': ltsv
+                    - null_value_pattern: '-'
+                    - keep_time_key: 'true'
+                    - label_delimiter: '='
+                    - delimiter_pattern: '/\s+(?=(?:[^"]*"[^"]*")*[^"]*$)/'
+                    - time_key: time
+                    - types: time:time
         - directive: source
           attrs:
-            - tag: cas.mitx.shibboleth.access
+            - '@id': cas_nginx_error_log
             - '@type': tail
             - enable_watch_timer: 'false'
-            - path: /var/log/apache2/mitx_shibboleth_access.log
-            - pos_file: /var/log/apache2/mitx_shibboleth_access.log.pos
-            - format: apache2
-            - time_format: '%d/%b/%Y:%H:%M:%S'
+            - tag: cas.nginx.error
+            - path: /var/log/nginx/error.log
+            - pos_file: /var/log/nginx/error.log.pos
+            - nested_directives:
+                - directive: parse
+                  attrs:
+                    - '@type': regexp
+                    - expression: '^(?<time>\d+\/\d+\/\d+\s\d+:\d+:\d+)\s(?<level_name>\[.*])\s(?<message>.*)client:\s(?<client_ip>\d{1,3}.\d{1,3}.\d{1,3}.\d{1,3}),\sserver:\s(?<server>.*),\srequest:\s"(?<method>.*)",\supstream:\s"(?<upstream>.*)",\shost:\s"(?<host>\d{1,3}.\d{1,3}.\d{1,3}.\d{1,3})"$'
```

Comment:
The expression does not seem to match the following error that I came across when sampling the nginx error log:
```2018/08/05 13:31:02 [crit] 30153#0: *1100 SSL_do_handshake() failed (SSL: error:1417D102:SSL routines:tls_process_client_hello:unsupported protocol) while SSL handshaking, client: 0.0.0.0, server: 0.0.0.0:443```

Note: changed client IP